src/main/resources/application.properties

```diff
@@ -22,10 +22,8 @@ osis.scality.vault.cache.listAccounts.expirationInMS=60000
 osis.scality.vault.cache.assumeRole.maxCapacity=1000
 osis.scality.vault.cache.assumeRole.expirationInMS=60000
 
-# S3 Config - Vault endpoint and S3 port with super admin credentials
+# S3 Config - Vault endpoint and S3 port
 osis.scality.s3.endpoint=http://localhost:8500
```

Comment:
Not sure why I cannot suggest but here we go:

```
# Vault S3 interface configuration
osis.scality.vault-internal-s3-interface.endpoint=http://localhost:8500
#osis.scality.admin.access-key=D4IT2AWSB588GO5J9T00
#osis.scality.admin.secret-key=UEEu8tYlsOGGrgf4DAiSZD6apVNPUWqRiPG0nTB6
```
Feel free to change the case of `vault-internal-s3--interface`. For Example: camel case `caultInternalS3Interface`


This is not S3 endpoint. Its an S3 Interface but vault service. Also, the keys are admin keys and not s3 keys. S3/Cloudserver doesn't understand admin routes.

S3 Endpoint port number is 8000 served by cloudserver.

Reply:
done in 916eec1d0143504a70d18803cb68a1a3643cabaa